Mozilla_Hubs_Privacy_Notice/en-US.md

```diff
@@ -0,0 +1,67 @@
+# Privacy Notice for Hubs and Spoke
+
+Version 3.2.3, July 11, 2019
```

Comment:
please add extra information on the release date using this format: https://github.com/mozilla/legal-docs/blob/master/firefox_privacy_notice/en-US.md